lib/slayer/version.rb

```diff
@@ -1,3 +1,3 @@
 module Slayer
-  VERSION = '0.4.0.beta4'
+  VERSION = '0.5.0.beta'
```

Comment:
We use the `beta` tag because `bundler` won't auto-upgrade to versions that are suffixed with alpha-numeric characters. We don't _know_ if anyone is still using the "stable" version, but we don't want to push a super breaking change without actually finishing out the documentation.